imports/plugins/included/payments-paypal/server/startup/startup.js

```diff
@@ -0,0 +1,9 @@
+import { Reaction, Hooks } from "/server/api";
+
+Hooks.Events.add("afterCoreInit", () => {
```

Comment:
ha, well, it seemed like the perfect opportunity. If we can set the `audience` for it, that is probably a better solution